.github/workflows/rust.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       - run: cargo test
       - run: cargo test --manifest-path fuzz/Cargo.toml
         if: ${{ matrix.rust }} == "stable"
+      - run: cargo test --benches
```

Comment:
Note that I did not simply extend the existing `cargo test` call, but instead added an additional `cargo test --benches`.

`cargo test --benches` executes some but not all unit tests in addition to all benchmark tests. See also https://github.com/rust-lang/cargo/issues/6454.